package.json

```diff
@@ -22,7 +22,9 @@
     "cypress": "cypress open",
     "cy:open": "cypress open --e2e --browser chrome",
     "cy:run": "cypress run --e2e --browser chrome",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
+    "create:openapi": "./create-openapi.sh",
+    "optic:capture": "dotenv -e .env optic capture openapi.yml --server-override $baseUrl --update interactive"
```

Comment:
I think you might want this to be run both ways:
- running `optic capture ...` without --update will check whether traffic matches your current spec (run this in CI)
- running it with `--update` will update the spec (run this locally)

Reply:
update locally, no update ci, got it